extensions/typescript-language-features/src/languageFeatures/quickFix.ts

```diff
@@ -51,6 +51,9 @@ class ApplyCodeActionCommand implements Command {
 	}
 }
 
+type ApplyFixAllCodeAction_args = {
```

Comment:
Types should be PascalCase  e.g. ApplyFixAllCodeAction_args -> ApplyFixAllCodeActionArgs

Reply:
As the maintainer of this code, I'm going with this name here :)